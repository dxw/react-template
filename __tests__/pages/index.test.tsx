import React from "react";
import { create } from "react-test-renderer";

import IndexPage from "../../pages/index";

it("renders correctly", () => {
  const component = create(<IndexPage />);

  expect(component.toJSON()).toMatchSnapshot();
});
