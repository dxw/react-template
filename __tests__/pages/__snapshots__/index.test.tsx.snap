// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <h1>
    Welcome to dxw's React template!
  </h1>,
  <style
    global={true}
    jsx={true}
  >
    
        body {
          font-family: "Lucida Sans", "Lucida Sans Regular", "Lucida Grande",
            "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
        }
      
  </style>,
]
`;
